geometry_msgs/Point CentorPoint
geometry_msgs/Vector3 DirectionVector
int32 lineNo
bool isStraight
